### PR TITLE
Ingestion: TAL 2006–2010

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

**Comment by anthology-assist** [comment]
@mcandito Could you review the previews of year 2006 - 2010?

**Comment by mcandito** [comment]
Hello @anthology-assist 
I identified a problem for [2007.tal-1](https://preview.aclanthology.org/tal-missing-ingestion/volumes/2007.tal-1),
the bug comes from my side, I'm sorry,
could you reingest:
http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2007.zip

The other volumes are ok
Thank you

**Comment by mcandito** [comment]
There are also 4 minor bugs:

- [2010.tal-2.1](https://preview.aclanthology.org/tal-missing-ingestion/2010.tal-2.1/) : the French title has a spurious ">" at its end, to be removed
- [2010.tal-2.5](https://preview.aclanthology.org/tal-missing-ingestion/2010.tal-2.5/) : the language in bib should be English, not French, and the title should simply be the English title : Transliteration as Alignment vs. Transliteration as Generation for Crosslingual Information Retrieval
- the English title of [2010.tal-3](https://preview.aclanthology.org/tal-missing-ingestion/volumes/2010.tal-3/) has a spurious "and" : [Opinions, sentiment and, and evaluative language] => [Opinions, sentiment, and evaluative language]
- spelling error in English title of [Résolution de métonymie des entités nommées : proposition d’une méthode hybride [Metonimy resolution for named entities: an hybrid approach]](https://preview.aclanthology.org/tal-missing-ingestion/2009.tal-1.4/) : metonimy => metonymy

**Comment by anthology-assist** [comment]
@mcandito Can you confirm that 2007.tal-1 is the only problematic volume? If possible I'd like to avoid re-ingest 2007.tal-2 and 2007.tal-3

**Comment by anthology-assist** [comment]
> There are also 4 minor bugs:
> 
> * [2010.tal-2.1](https://preview.aclanthology.org/tal-missing-ingestion/2010.tal-2.1/) : the French title has a spurious ">" at its end, to be removed
> * [2010.tal-2.5](https://preview.aclanthology.org/tal-missing-ingestion/2010.tal-2.5/) : the language in bib should be English, not French, and the title should simply be the English title : Transliteration as Alignment vs. Transliteration as Generation for Crosslingual Information Retrieval
> * the English title of [2010.tal-3](https://preview.aclanthology.org/tal-missing-ingestion/volumes/2010.tal-3/) has a spurious "and" : [Opinions, sentiment and, and evaluative language] => [Opinions, sentiment, and evaluative language]
> * spelling error in English title of [Résolution de métonymie des entités nommées : proposition d’une méthode hybride [Metonimy resolution for named entities: an hybrid approach]](https://preview.aclanthology.org/tal-missing-ingestion/2009.tal-1.4/) : metonimy => metonymy

done!

**Comment by mcandito** [comment]
Yes I confirm 2007.tal-1 is the only one to reingest
Thanks for the debugs!

**Comment by mjpost** [comment]
@anthology-assist Let me know when you've checked off these items and I'll review.

**Comment by anthology-assist** [comment]
@mcandito Could you check 2007.tal-1 again?

**Comment by mcandito** [comment]
Hello,

2007.tal-1 is fine except for two minor bugs:

There are two articles in English that are shown with the French title due to the previous mismatch

- for https://preview.aclanthology.org/tal-missing-ingestion/2007.tal-1.6/ the title should be :
Evaluation in Natural Language Generation: Lessons from Referring Expression Generation

- for https://preview.aclanthology.org/tal-missing-ingestion/2007.tal-1.8/ the title should be :
Prosodic Phrase Break Prediction: Problems in the Evaluation of Models against a Gold Standard

The languages are correctly set though
Thank you

File: data/xml/2007.tal.xml [hunk 1]
```diff
@@ -0,0 +1,240 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2007.tal">
+  <volume id="1" ingest-date="2023-07-25">
+    <meta>
+      <booktitle>Traitement Automatique des Langues, Volume 48, Numéro 1 : Principes de l'évaluation en Traitement Automatique des Langues [Principles of Evaluation in Natural Language Processing</booktitle>
```
Comment (mjpost) [inline data/xml/2007.tal.xml:5]: Missing closing bracket

File: data/xml/2008.tal.xml [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+      <author><first>Masayuki</first><last>Asahar</last></author>
+      <author><first>Yuji</first><last>Matsumoto</last></author>
+      <pages>271–293</pages>
+      <url hash="7e61b573">2008.tal-2.11</url>
+      <bibkey>evans-etal-2008-cocytus</bibkey>
+    </paper>
+  </volume>
+  <volume id="3" ingest-date="2023-07-18">
+    <meta>
+      <booktitle>Traitement Automatique des Langues, Volume 49, Numéro 3 : Recherches actuelles en phonologie et en phonétique : interfaces avec le traitement automatique des langues [Current Research in Phonology and Phonetics: Interfaces with Natural-Language Processing.]</booktitle>
```
Comment (mjpost) [inline data/xml/2008.tal.xml:197]: `Processing.` → `Processing` (remove period

File: data/xml/2007.tal.xml (hunk 1 above)
Comment (anthology-assist) [reply data/xml/2007.tal.xml:5]: Done

File: data/xml/2008.tal.xml (hunk 2 above)
Comment (anthology-assist) [reply data/xml/2008.tal.xml:197]: Done

**Comment by anthology-assist** [comment]
@mjpost should be ready to merge!

**Comment by anthology-assist** [comment]
> Hello,
> 
> 2007.tal-1 is fine except for two minor bugs:
> 
> There are two articles in English that are shown with the French title due to the previous mismatch
> 
> * for https://preview.aclanthology.org/tal-missing-ingestion/2007.tal-1.6/ the title should be :
>   Evaluation in Natural Language Generation: Lessons from Referring Expression Generation
> * for https://preview.aclanthology.org/tal-missing-ingestion/2007.tal-1.8/ the title should be :
>   Prosodic Phrase Break Prediction: Problems in the Evaluation of Models against a Gold Standard
> 
> The languages are correctly set though Thank you

Done!

**Review by mjpost — APPROVED** [review]

**Pull request merged**